src/components/Legend/Legend.js

```diff
@@ -5,7 +5,7 @@ import style from './Legend.scss';
 const Legend = props => (
   <div className={props.className}>
     {props.title && <div className={style.title}>{props.title}</div>}
-    {props.series.map(item => (
+    {props.series.reverse().map(item => (
```

Comment:
`.reverse()` mutates the original array and mutating props is a no-no (it changes the sort order in other components - potentially, depending on rendering order and who knows what else).

So you first need to clone the array then sort the new array. Check out `sortChartData()` in `CityColumnChart.js` for an example (`.sort()` also mutates an array in place)

Actually, why reverse this anyway? Wouldn't we want the legend to be in the same order that the indicators are listed in the data? This is without sorting, seems the right way round to me:

![image](https://user-images.githubusercontent.com/4443482/32151602-4d1646ce-bd72-11e7-876d-9f3a9934385e.png)

But if you want it to sort differently on mobile, where it looks like this:
![image](https://user-images.githubusercontent.com/4443482/32151621-6fe7b534-bd72-11e7-91de-26814dfb0d47.png)


You should use `flex-direction: column-reverse` to do so at that screen size only.

Reply:
Huh! I didn't know JS arrays' `reverse()` method did that.

On stacked, it's much of a muchness, but the reason to reverse the order becomes clear when you look at grouped (rather than stacked) charts.

Reply:
Yep, it's a mixed bag which methods mutate the original object in JavaScript. I had to google `reverse()` to check :)

FYI do you know about `legendIndex` in Highcharts? You can control the order that way, but it's probably more complex.